content-src/experiments/dev-example.yaml

```diff
@@ -0,0 +1,89 @@
+dev: true
```

Comment:
Do we need to add this new key to the experiment template?
https://github.com/mozilla/testpilot/blob/master/docs/experiments/experiment-content/template.yaml